NewSpring/ApollosProject
# (DEV-4161) Fixes Empty Communicator Bug

## DESCRIPTION

<img width="475" alt="Screen Shot 2019-10-21 at 10 29 38 PM" src="https://user-images.githubusercontent.com/2659478/67255957-a983e600-f452-11e9-9b03-5f1dcc428292.png">


### What does this PR do, or why is it needed?

When `communicator` was missing, it was breaking the sermon notes.

### How do I test this PR?

Pull up the "Pornography" sermon in the app. And check one that has a communicator too.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
